pkg/kv/kvclient/kvcoord/dist_sender_test.go

```diff
@@ -5130,6 +5134,188 @@ func TestSendToReplicasSkipsStaleReplicas(t *testing.T) {
 	}
 }
 
+// Test that DistSender.computeReadAndWriteRUMultipliers() returns the right
+// computed read and write RU multipliers.
+func TestDistSenderComputeRUMultipliers(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	ctx := context.Background()
+	stopper := stop.NewStopper()
+	defer stopper.Stop(ctx)
+
+	clock := hlc.NewClockWithSystemTimeSource(time.Nanosecond /* maxOffset */)
+	rpcContext := rpc.NewInsecureTestingContext(ctx, clock, stopper)
+	rddb := MockRangeDescriptorDB(func(key roachpb.RKey, reverse bool) (
+		[]roachpb.RangeDescriptor, []roachpb.RangeDescriptor, error,
+	) {
+		// computeReadAndWriteRUMultipliers should not be using this at all,
+		// but DistSender insists on having a non-nil one.
+		return nil, nil, errors.New("range desc db unexpectedly used")
+	})
+	st := cluster.MakeTestingClusterSettings()
+
+	// Set regional cost multiplier table.
+	//                     | us-east1 | eu-central1 | asia-southeast1
+	//     -----------------------------------------------------------
+	//        us-east1     |    1     |     1.5     |        2
```

Comment:
nit: the table in this comment does not match the table in the configuration.

(us-east, asia-southeast1) should be 2.6 instead of 2.